packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import OrgNav from './org-nav/index';
+import ProjectNav from './project-nav/index';
+import {
+  DataInterface,
+  Product,
+  URLSInterface,
+  HostsInterface,
+  NavItem,
+} from './types';
+
+interface MongoNavInterface {
+  activeProduct: Product;
+  activeNav?: NavItem;
+  admin?: boolean;
+  data: DataInterface;
+  onOrganizationChange: React.ChangeEventHandler;
+  onProjectChange: React.ChangeEventHandler;
+  constructOrganizationURL?: (orgID: string) => string;
+  constructProjectURL?: (orgID: string, projID: string) => string;
+  showProjNav?: boolean;
+  hosts?: HostsInterface;
+  urls?: URLSInterface;
+}
+
+export default function MongoNav({
+  activeProduct,
+  activeNav,
+  data,
+  hosts,
+  onOrganizationChange,
+  onProjectChange,
+  urls,
+  showProjNav = true,
+  admin = false,
+  constructOrganizationURL = orgId =>
+    `https://cloud.mongodb.com/v2#/org/${orgId}/projects`,
```

Comment:
It'd be nice if these default functions could default to the hosts value for constructing these URLs. Would allow consumers to only provide the hosts value.

Reply:
Hm so just accept `(host, orgId)` and then we construct?

Reply:
👍